ssd/online/R3BFootOnlineSpectra.cxx

```diff
@@ -120,62 +115,60 @@ InitStatus R3BFootOnlineSpectra::Init()
 
     // MAIN FOLDER-FOOT
     TFolder* mainfol = new TFolder("Foot", "Foot-Si info");
-    // Folder for mapped data
     TFolder* mapfol = new TFolder("Map", "Map FOOT info");
-    mainfol->Add(mapfol);
-    // Folder for cal data
     TFolder* calfol = new TFolder("Cal", "Cal FOOT info");
-    // Folder for hit data
     TFolder* hitfol = new TFolder("Hit", "Hit FOOT info");
+    mainfol->Add(mapfol);
 
-    //  CANVAS 1  -------------------------------
-    // auto cMap = new TCanvas("FOOT_mapped", "mapped info", 10, 10, 500, 500);
-    // cMap->Divide(fNbDet / 2, 2);
-    // cMap->Divide(3, 2);
+    auto cMap = new TCanvas("FOOT_mapped", "mapped info", 10, 10, 500, 500);
+    cMap->Divide(4, 3);
+    mapfol->Add(cMap);
 
-    // Mapped data
+    //================  Mapped data =====================
     fh2_EnergyVsStrip.resize(fNbDet);
     int i_pad = 1; // pad iterator
     for (Int_t i = 0; i < fNbDet; i++)
     { // one histo per detector
         sprintf(Name1, "fh2_energy_vs_strip_det_%d", i + 1);
         sprintf(Name2, "Mapped energy vs strip number for FOOT Det: %d", i + 1);
-        auto cMap = new TCanvas(Name1, Name2, 10, 10, 500, 500);
         fh2_EnergyVsStrip[i] = new TH2F(Name1, Name2, 640, 1, 641, binsE, minE, maxE);
         fh2_EnergyVsStrip[i]->GetXaxis()->SetTitle("Strip number");
         fh2_EnergyVsStrip[i]->GetYaxis()->SetTitle("Energy [channels]");
         fh2_EnergyVsStrip[i]->GetYaxis()->SetTitleOffset(1.4);
         fh2_EnergyVsStrip[i]->GetXaxis()->CenterTitle(true);
-        fh2_EnergyVsStrip[i]->GetYaxis()->CenterTitle(true);
-        // int foot_num = i+1;
-        // if(foot_num==2 || foot_num==4 || foot_num==7
-        //		|| foot_num==9 || foot_num==10 || foot_num==11)
-        //{
-        // cMap->cd(i_pad);
-        fh2_EnergyVsStrip[i]->Draw("czol");
-        for (int i_asic = 1; i_asic < 10; i_asic++)
+        int foot_num = i+1;
+        if(foot_num==1 || foot_num==2 || foot_num==13 
+                || foot_num==4 || foot_num==11 || foot_num==6
+                || foot_num==7 || foot_num==12 || foot_num==9
+                || foot_num==10 || foot_num==15 || foot_num==16)
         {
-            TLine* l = new TLine(64.5 * i_asic, minE, 64.5 * i_asic, maxE);
-            l->SetLineStyle(7);
-            l->SetLineWidth(2);
-            l->SetLineColor(2);
-            l->Draw();
+            cMap->cd(i_pad);
+            fh2_EnergyVsStrip[i]->Draw("col");
+            for(int i_asic=1; i_asic<10; i_asic++)
+            {
+                TLine* l = new TLine(64.5*i_asic,minE,64.5*i_asic,maxE);
+                l->Draw("same");
+                l->SetLineStyle(7);
+                l->SetLineWidth(1);
+                l->SetLineColor(13);
+            }
+            i_pad++;
         }
-        mapfol->Add(cMap);
-        // i_pad++;
-        //}
+        mapfol->Add(fh2_EnergyVsStrip[i]);
```

Comment:
Doing this (Histograms -> Folder), the canvas cMap does nothing, please compare to my previous PR. It should be
Histograms -> Canvas -> Folder
In my previous PR I added one canvas per detector, otherwise the figures are too small

Reply:
done